content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -93,16 +105,42 @@ reproduces the same set of steps that the kubelet performs to calculate
 file-backed memory on the inactive LRU list) from its calculation, as it assumes that
 memory is reclaimable under pressure.
 
-The kubelet recognizes two specific filesystem identifiers:
+The kubelet recognizes three specific filesystem identifiers:
+
+1. `nodefs`: The node's main filesystem, used for local disk volumes,
+    emptyDir volumes not backed by memory, log storage, ephemeral storage,
+    and more. For example, `nodefs` contains `/var/lib/kubelet`.
+
+1. `imagefs`: An optional filesystem that container runtimes can use to store
+   container images (which are the read-only layers) and container writable
+   layers.
+
+1. `containerfs`: An optional filesystem that container runtime can use to
+   store the writeable layer, similarly to what the main filesystem would (see
+   `nodefs`) aside from the container images, which can then be separated (or
+   split from, if you wish) such that the `imagefs` filesystem would only
+   store the images (read-only layers) and nothing else.
+
+As such, kubelet generally allows three options for container filesystems:
 
-1. `nodefs`: The node's main filesystem, used for local disk volumes, emptyDir
-   volumes not backed by memory, log storage, and more.
-   For example, `nodefs` contains `/var/lib/kubelet/`.
-1. `imagefs`: An optional filesystem that container runtimes use to store container
-   images and container writable layers.
+- Everything is on the single `nodefs`, also referred to as "rootfs" or
+  simply "root", and there is no dedicated image filesystem.
 
-Kubelet auto-discovers these filesystems and ignores other node local filesystems. Kubelet
-does not support other configurations.
+- Container storage (see `nodefs`) is on a dedicated disk, and `imagefs`
+  (writable and read-only layers) is separate from the root filesystem.
+  This is often referred to as "Split Disk" (or "Separate Disk") filesystem.
+
+- Container filesystem `containerfs` (same as `nodefs` plus writable
+  layers) is on root and the container images (read-only layers) are
+  stored on separate `imagefs`. This is often referred to as "Split Image"
+  filesystem.
+
+Currently, multiple filesystems for images and containers are not supported.
+
+The kubelet will attempt to auto-discover these filesystems and their
+current configuration directly from the underlying container runtime and,
+as such, ignore other local node filesystems; kubelet does not support
+different configurations at the moment.
```

Comment:
This sentence is confusing. I have no idea what "different configurations" means.

Reply:
This sentence is part of the text that was there per:

- https://github.com/kubernetes/website/commit/c184f20f76eaf58ece0ef14e025da1a4fa9d4f0d#diff-56de0c49fa24aa21138c65e0b8a9d71ab70f48159fc8184d9fee6e82793418a2R85-R86

I am not sure what precisely the original author had in mind. I can only assume, that "other" or "different" configurations refer to anything that has not been auto-discovered from the container runtime.

Reply:
We can drop this sentence or expand upon it. Also, perhaps @shannonxtreme recalls what was meant to be conveyed here?

Reply:
Done.